Funcky/Monads/Reader/Reader.Transformation.cs

```diff
@@ -1,10 +0,0 @@
-namespace Funcky.Monads;
-
-public static partial class ReaderExtensions
-{
-    [Pure]
```

Comment:
Why are there no Reader-Extensions?

Reply:
Reader itself is not `Traversable`. I moved this extension method to `EnumerableExtensions`